specification/databox/resource-manager/readme.md

```diff
@@ -26,25 +26,25 @@ These are the global settings for the DataBox API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2018-01
+tag: package-2019-09
 ```
 
-### Tag: package-2018-01
+### Tag: package-2019-09
```

Comment:
I think you should add package-2019-09 at the top instead of replacing package-2018-01.

Reply:
Hi @jsntcy It's in basic information block and I think we should replace it.

Reply:
Sorry, I mislooked it.

---
In reply to: [377524675](https://github.com/Azure/azure-rest-api-specs/pull/8355#discussion_r377524675) [](ancestors = 377524675)